src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -52,7 +52,7 @@ namespace Npgsql
 #pragma warning disable CA1010
     public abstract class NpgsqlDataReader : DbDataReader
 #pragma warning restore CA1010
-#if NETSTANDARD1_3
+#if NETSTANDARD1_3 || NETSTANDARD2_0
```

Comment:
Can we please inverse this check, i.e.:
```c#
#if !NET45 && !NET451
```

The advantage would be that any targeted future version of .NET Standard will automatically have this support, and we won't have this issue again... If we target a new .NET Framework we'll get a compilation error there (because IDbColumnSchemaGenerator doesn't exist), so it'll be very easy to detect and fix.

Reply:
Sure thing and good thought - I'll push that up tomorrow.